src/hotspot/share/opto/superword.cpp

```diff
@@ -2897,14 +2763,13 @@ bool SuperWord::output() {
   }
 
   for (int i = 0; i < _block.length(); i++) {
-    Node* n = _block.at(i);
+    Node* n = _block.at(i); // last in pack
```

Comment:
Nit: how about moving this comment line to the upward side of the if clause on L2768? A little bit confusing now.